core/src/main/java/org/apache/accumulo/core/metadata/schema/MetadataSchema.java

```diff
@@ -263,6 +265,14 @@ public static String getRowPrefix() {
       return section.getRowPrefix();
     }
 
+    public static String encodeRow(String value) {
+      return getRowPrefix() + SortSkew.getCode(value) + value;
```

Comment:
Could the getRowPrefix method be made private?  Does anything use it externally?

Reply:
Actually removed it.   Used to be used in SimpleGarbageCollector but that has all changed.  Consolidated length constant into below which is where it belonged.
```java
 public static class DeletesSection {
    private static final Section section =
        new Section(RESERVED_PREFIX + "del", true, RESERVED_PREFIX + "dem", false);

    private static final int encoded_prefix_length =
        section.getRowPrefix().length() + SortSkew.SORTSKEW_LENGTH;

    public static Range getRange() {
      return section.getRange();
    }

    public static String encodeRow(String value) {
      return section.getRowPrefix() + SortSkew.getCode(value) + value;
    }

    public static String decodeRow(String row) {
      return row.substring(encoded_prefix_length);
    }

  }
```

Reply:
nice